components/contribution-flow/ContributionSummary.js

```diff
@@ -116,7 +116,7 @@ const ContributionSummary = ({
               </AmountLine>
             ))}
 
-          {Boolean(platformTip) && (
+          {Boolean(platformTip) && !isCrypto && (
```

Comment:
Why do this if platformTip is already set to zero?

Reply:
Ops, this is a mistake. I've remove this condition.